packages/z/zeek/package.yml

```diff
@@ -0,0 +1,46 @@
+name       : zeek
+version    : 6.0.2
+release    : 1
```

Comment:
I'd rather not have "replacing bro" in the commit message, though it can stay in the PR name. Please put it in the summary instead.